### DISCUSSION TITLE
Use a localization platform

### DISCUSSION THREAD

**Discussion by vachan-maker** [hoppscotch/hoppscotch#1394] [Ideas]
Hoppscotch should use a localization management platform for managing translations.
These are some of the most commonly used localization management systems

[Crowdin](https://crowdin.com)
[Weblate](https://weblate.org/en/)
[Transifex](https://transifex.com)

Crowdin is pretty good and it has a really interface as well.

- Translators will receive notifications when new strings have been added.
- They can ask question and have discussions within the platform.
- A really nice and friendly interface.
- Collaboration among users.
- Multiple languages can be added.
- Project will probably get some more attention and bring in new users.

Now, I am not a programmer but I am pretty sure that devs will find this easy as pretty much all these platforms integrate with Github. So, PRs can automatically be opened and the translations will be added as commits in the PR. There are other ways to do this. As I said earlier, I really don't know the technical side of using such a platform. But I am pretty sure you will find this helpful.

Thanks.

**Comment by AndrewBastin** [comment]
+1

We should really do something like that.

@liyasthomas
